src/components/MaterialUI/DataDisplay/PredefinedElements/Placeholder.js

```diff
@@ -20,11 +21,55 @@ const useStyles = makeStyles(theme => ({
 		fontSize: theme.typography.h3Size,
 		color: theme.palette.grey.icon,
 	},
+	placeholderRow: {
+		alignSelf: "flex-start",
+		width: "100%",
+		display: "flex",
+		borderBottom: `1px solid ${theme.palette.grey.borders}`,
+		backgroundColor: `${theme.palette.background.paper}!important`,
+	},
+	placeholderCell: {
+		display: "inline-flex",
+		padding: theme.spacing(2.7, 1.6),
+		paddingRight: theme.spacing(4.6),
+		flex: 1,
+		alignItems: "center",
+	},
+	textSkeleton: {
+		width: "100%",
+		height: 6,
+		animation: false,
+	},
+	chipSkeleton: {
+		width: 60,
+		height: 25,
+		animation: false,
+		borderRadius: "15px",
+	},
 }));
 
-const Placeholder = ({ icon, title, subtitle }) => {
+const Placeholder = ({ icon, title, subtitle, cellList = [] }) => {
 	const classes = useStyles();
-	return (
+	return cellList.length ? (
```

Comment:
From this logic I see that when we have cellList specified, then Icon and Title will not be rendered. We should be able to render all together  when specifiend like this:
![image](https://user-images.githubusercontent.com/6649972/105957579-1f2d2780-6082-11eb-9a00-647a5b19f2b5.png)